src/main/java/build/buildfarm/CASMemoryProfileClient.java

```diff
@@ -0,0 +1,74 @@
+package build.buildfarm;
```

Comment:
Drop this file and add the functionality to Cat.java, adding the necessary methods to Instance, StubInstance, and the server instances.